src/pkg/services/m365/api/graph/service_test.go

```diff
@@ -243,24 +242,21 @@ func (suite *GraphIntgSuite) TestAdapterWrap_retriesConnectionClose() {
 		suite.credentials.AzureClientID,
 		suite.credentials.AzureClientSecret,
 		count.New(),
-		appendMiddleware(&alwaysECONNRESET))
+		appendMiddleware(&alwaysECONNRESET),
+		// Configure retry middlewares so that they don't retry on connection reset.
+		// Those middlewares have their own tests to verify retries.
+		MaxRetries(-1))
 	require.NoError(t, err, clues.ToCore(err))
 
-	// Set retry delay to a low value to reduce test runtime.
+	// Retry delay doesn't really matter here since all requests will be intercepted
+	// by the test middleware. Set it to 0 to reduce test runtime.
 	aw := adpt.(*adapterWrap)
-	aw.retryDelay = 10 * time.Millisecond
-
-	// the query doesn't matter
-	_, err = users.NewItemCalendarsItemEventsDeltaRequestBuilder(url, adpt).Get(ctx, nil)
```

Comment:
I've removed this API call since it wasn't adding much value. We are testing retries with another API call right below.